github_connector/readme/USAGE.rst

```diff
@@ -84,6 +84,34 @@ Finally, you can download locally the source code of all your branches:
 
    .. image:: ../static/description/github_repository_branch_list.png
 
+Download a private repository
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you want to fetch the source code from a private repository, you need a token (instead of login/password) with the right scopes:
+
+#. Go to 'Github Account' / 'Settings' / 'Developer settings' / 'Personal access tokens' / 'Generate new token'.
+
+   .. image:: https://raw.githubusercontent.com/OCA/interface-github/12.0/github_connector/static/description/generate_new_token.png
```

Comment:
Use this to avoid having to update the link manually on each version bump:

```suggestion
   .. image:: ../static/description/generate_new_token.png
```

The bot will convert that to an absolute path when building the README.